airflow/models/connection.py

```diff
@@ -271,12 +271,6 @@ def _create_host(protocol, host) -> str | None:
 
     def get_uri(self) -> str:
         """Return connection in URI format."""
-        if self.conn_type and "_" in self.conn_type:
```

Comment:
Congratulations on you first PR. 
I believe this is still needed. Let't wait for others comments.

Reply:
Hey @dirrao - any recommendations for getting other eyes on this?